src/calliope/backend/parsing.py

```diff
@@ -855,8 +855,7 @@ def raise_caught_errors(self):
             exceptions.print_warnings_and_raise_errors(
                 errors={f"{self.name}": self._errors},
                 during=(
-                    "math string parsing (marker indicates where parsing stopped, "
-                    "which might not be the root cause of the issue; sorry...)"
+                    "math string parsing (marker indicates where parsing stopped, but may not point to the root cause of the issue)"
```

Comment:
who doesn't wrap lines in their IDEs these days?

Reply:
Huh, apparently me! Thanks for the tip.

Although, I'd still recommend to do this, because code isn't always evaluated through IDEs (GitHub reviews, for example).

I'll mark this as resolved since it does not trigger a linter issue, and is just style.